plugins/woocommerce-admin/client/customize-store/assembler-hub/layout.tsx

```diff
@@ -66,119 +68,136 @@ export const Layout = () => {
 	const [ backgroundColor ] = useGlobalStyle( 'color.background' );
 	const [ gradientValue ] = useGlobalStyle( 'color.gradient' );
 
+	const { record: template } = useEditedEntityRecord();
+	const { id: templateId, type: templateType } = template;
+
 	return (
 		<LogoBlockContext.Provider
 			value={ {
 				logoBlock,
 				setLogoBlock,
 			} }
 		>
-			<div className={ classnames( 'edit-site-layout' ) }>
-				<motion.div
-					className="edit-site-layout__header-container"
-					animate={ 'view' }
+			<EntityProvider kind="root" type="site">
```

Comment:
This diff is hard to read but all I did was move EntityProvider here so we can access blocks from inside side nav